docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/configuration-description.md

```diff
@@ -0,0 +1,95 @@
+# Configuration Description
+
+## Meta Service
+
+Property Name               | Default Value            | Description
+--------------------------- | ------------------------ | -----------
```

Comment:
Please keep the same style for the description field. Whether a period is needed for each description?

Reply:
fixed